### ISSUE TITLE
subctl verify reports failure if an AfterSuite fails

### ISSUE THREAD

**Issue by skitt** [submariner-io/submariner-operator#1900]
> Failure [0.000 seconds]
> [AfterSuite] AfterSuite 
> github.com/submariner-io/shipyard@v0.12.0-rc0/test/e2e/e2e.go:55
> 
>   Failed to perform patch operation
>   Unexpected error:
>       <*errors.StatusError | 0xc00062f040>: {
>           ErrStatus: {
>               TypeMeta: {Kind: "", APIVersion: ""},
>               ListMeta: {
>                   SelfLink: "",
>                   ResourceVersion: "",
>                   Continue: "",
>                   RemainingItemCount: nil,
>               },
>               Status: "Failure",
>               Message: "nodes \"ip-10-0-2-2\" not found",
>               Reason: "NotFound",
>               Details: {Name: "ip-10-0-2-2", Group: "", Kind: "nodes", UID: "", Causes: nil, RetryAfterSeconds: 0},
>               Code: 404,
>           },
>       }
>       nodes "ip-10-0-2-2" not found
>   occurred
> 
>   github.com/submariner-io/shipyard@v0.12.0-rc0/test/e2e/framework/framework.go:513
> ------------------------------
> 
> Ran 24 of 41 Specs in 612.977 seconds
> FAIL! -- 24 Passed | 0 Failed | 0 Pending | 17 Skipped
> [] E2E failed
> 
> subctl version: v0.12.0-rc0

This particular clean-up probably shouldn’t report an error if it fails to find the resource it’s trying to remove.

**Comment by mkolesnik** [comment]
It's not clear where the error happened, do you have more logs or is this all there is?

**Comment by skitt** [comment]
> It's not clear where the error happened, do you have more logs or is this all there is?

I don’t have any more logs, but I think this is happening in `GatewayCleanup`.

**Comment by skitt** [comment]
> Also seems a bit weird that it didn't find the node? My guess is that it's transient (and rare) since we don't see it in CI so I wonder what was going on at the time, was cleanup running?

Yes, I’m not sure why this happened; the main issue I have is that failing to unlabel the node shouldn’t fail if the node no longer exists.

**Comment by skitt** [comment]
> > Also seems a bit weird that it didn't find the node? My guess is that it's transient (and rare) since we don't see it in CI so I wonder what was going on at the time, was cleanup running?
> 
> Yes, I’m not sure why this happened; I wasn’t running cleanup in parallel. The main issue I have is that failing to unlabel the node shouldn’t fail if the node no longer exists.
